third_party/libunwindstack/MapInfo.cpp

```diff
@@ -115,6 +118,19 @@ Memory* MapInfo::GetFileMemory() {
     return nullptr;
   }
 
+  if (IsPotentiallyPeCoffFile(name())) {
+    // Always map the whole PE, as we always need the headers.
+    // This is similar to "No elf at offset, try to init as if the whole file is an elf" below.
+    if (memory->Init(name(), 0)) {
+      set_object_offset(offset());
+      // Keep this simple for PEs: assume that the headers are always at the beginning of the file
```

Comment:
I think it is, and the evidence is that the PE format always has the headers at the beginning of the file. From what I understand here, the ELF case is much more complicated because cases different than a plain ELF file mapped into memory are considered (e.g., see the existing comments below).

Reply:
I expanded this comment a bit.